exonum/examples/sample_runtime.rs

```diff
@@ -156,15 +140,30 @@ impl Runtime for SampleRuntime {
             .get(&call_info.instance_id)
             .ok_or(SampleRuntimeError::IncorrectCallInfo)?;
 
+        if context.interface_name == Initialize::NAME {}
+
         println!(
-            "Executing method {} of service {}",
-            call_info.method_id, call_info.instance_id
+            "Executing method {}#{} of service {}",
+            context.interface_name, call_info.method_id, call_info.instance_id
         );
 
-        // Simple transaction executor.
-        match call_info.method_id {
+        const SERVICE_INTERFACE: &str = "";
+        match (context.interface_name, call_info.method_id) {
```

Comment:
Documentation of `execute` method is now not sufficient. Please expand on:
1. Requirements of handling Initialize interface by Runtimes.
2. Default interface name
3. Requirements on error handling in case of invalid runtime name passed.